src/hw1/bash.py

```diff
@@ -0,0 +1,39 @@
+def head(filename="first.txt", lines=10):
+    if line < 0:
+        raise RuntimeError("Number of lines must be positive")
+    with open(filename, "rt") as file:
+        file_head = [file.readline() for i in range(lines)]
+    print("".join(file_head))
+
+
+def tail(filename="first.txt", lines=10):
+    if line < 0:
+        raise RuntimeError("Number of lines must be positive")
+    with open(filename, "rt") as file:
+        file_backward = [line for line in reversed(file.readlines())]
+        file_backward[0] += "\n"
+        file_backward[-1] = file_backward[-1][0:-1]
```

Comment:
I do not understand the point of these actions. BTW, did you test your code? When I try to run it, it skips the final line.